internal/mempool/mempool.go

```diff
@@ -1116,18 +1116,24 @@ func (mp *TxPool) fetchInputUtxos(tx *dcrutil.Tx, isTreasuryEnabled bool) (*bloc
 	}
 
 	// Attempt to populate any missing inputs from the transaction pool.
-	for originHash, entry := range utxoView.Entries() {
-		if entry != nil && !entry.IsFullySpent() {
+	for _, txIn := range tx.MsgTx().TxIn {
+		prevOut := &txIn.PreviousOutPoint
```

Comment:
I renamed `prevOut` to `outpoint` in the cases where the variable referred to an output of a transaction, as opposed to a previous outpoint associated with a transaction input (I found the naming `prevOut` to be misleading in that case).  The cases where it was actually referring to a previous outpoint I left as is.  Given that, do you still think it would be best to rename all of them to `outpoint`?  I don't have a strong opinion and would be good with it either way.

Reply:
Makes sense.  I similarly don't feel strongly either way myself.  I suppose it really just depends on how it's going to be used as outpoints ultimately become prev outs, so if this make it more clear to you, works for me!

Reply:
Sounds good.  In that case, I'll leave it as is since I think it's _slightly_ more clear, especially in the cases where a function is dealing with both the inputs and outputs of a given transaction.